tests/integration/dynamic_ip_test.py

```diff
@@ -414,7 +415,9 @@ def test_ipv6_dhcp_off_and_option_on(dhcpcli_up):
 
     libnmstate.apply(desired_state)
 
-    dhcp_cli_current_state = statelib.show_only((DHCP_CLI_NIC,))[INTERFACES][0]
+    dhcp_cli_current_state = statelib.show_only((DHCP_CLI_NIC,))[
```

Comment:
Same here, please try to improve the formatting.

Reply:
Fixed!